client/components/Users.js

```diff
@@ -83,12 +74,20 @@ export class Users extends React.Component {
                     {
                     this.state.users.map(user => {
                         return (
-                            <tr key={++id}>
-                                <td>{user}</td>
-                                <td>{`${user}@email.com`}</td>
+                            <tr key={user.id}>
+                                <td>{user.name}</td>
+                                <td>{user.email}</td>
                                 <td><button onClick={this.handleResetPassword}>Reset</button></td>
-                                <td><button name={user} value={id} onClick={this.handleDeleteUser}>X</button></td>
-                                <td><input name={user} type="checkbox" onChange={this.handleChecked} /></td>
+                                <td><button name={user.name} value={user.id} onClick={() => {
+                                     if(this.state.adminUsers.includes(user.name)){
+                                         alert('Change admin status before deleting')
+                                     } else {
+                                         console.log(`${user.name} deleted`)
+                                         const updatedUsers = this.state.users.filter( u => u.name !== user.name)
+                                         this.setState({users: updatedUsers})
+                                     }
+                                    this.props.deleteUser(user)}}>X</button></td>
+                                <td><input name={user.name} type="checkbox" onChange={this.handleChecked} /></td>
```

Comment:
this doesn't show whether user is actually admin

Reply:
Yup, will have this done today